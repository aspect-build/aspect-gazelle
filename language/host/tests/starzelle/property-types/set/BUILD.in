# gazelle:s foo
# gazelle:s_defaults asdf

# gazelle:sa foo, bar
# gazelle:sa_defaults asdf, fdsa

# gazelle:b foo
# gazelle:b_defaults foo, bar

# gazelle:n 123456
# gazelle:n_defaults foo, 5432
