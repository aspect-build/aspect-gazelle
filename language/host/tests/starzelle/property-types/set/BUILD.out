load("@test//:x.bzl", "props_test")

# gazelle:s foo
# gazelle:s_defaults asdf

# gazelle:sa foo, bar
# gazelle:sa_defaults asdf, fdsa

# gazelle:b foo
# gazelle:b_defaults foo, bar

# gazelle:n 123456
# gazelle:n_defaults foo, 5432

props_test(
    name = "no_defaults",
    b = False,
    n = 123456,
    s = "foo",
    sa = ["foo, bar"],
)

props_test(
    name = "with_defaults",
    b = False,
    n = 123,
    s = "asdf",
    sa = ["asdf, fdsa"],
)
